flutter/assets/tasks.pbtxt

```diff
@@ -303,18 +303,18 @@ task {
     type: COCOGEN
     full {
       name: "COCO validation set for Stable Diffusion"
-      input_path: "https://github.com/anhappdev/tmp/releases/download/5/coco_gen_full.tfrecord"
+      input_path: "https://github.com/anhappdev/tmp/releases/download/5/coco_gen_test.tfrecord"
```

Comment:
Should it be?
* full -> coco_gen_full (5000 prompts)
* lite -> coco_gen_test (150 prompts)

Reply:
I think that's what was decided right? We will use some random samples from full set for performance test.
But use the 150 prompts for measuring accuracy.
But we can double check in the meeting today.

Reply:
Yes. You are correct. I forget that we use the lite one for performance test.